macros/edr/data_monitoring/data_monitors_configuration/get_buckets_configuration.sql

```diff
@@ -1,38 +1,68 @@
-{% macro get_min_bucket_start(metric_properties) %}
-    {% do return((elementary.get_run_started_at() - modules.datetime.timedelta(metric_properties.days_back | int)).strftime("%Y-%m-%d 00:00:00")) %}
+{# This macro can't be used without truncating to full buckets #}
```

Comment:
Good comment. I'm curious:
- what exactly would happen if the buckets passed into it are not truncated
- is there a way to enforce it instead of hoping the devs that will use the macro will read the comment? (since Jinja is hard maybe at least renaming the macro so it hints this ?)